client/components/Thumbnail.jsx

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 
 // border compares activeIndex & id to see
 // whether it should have a purple border
 const Thumbnail = styled.img`
-  border: ${props => (props.activeIndex === props.id
+  border: ${(props) => (props.activeIndex === props.id
     ? '2px solid #7f187f'
     : '2px solid white')};
```

Comment:
Too long for a single line? - Love the usage here by the way!

Reply:
Not really. I just found it cleaner.